### PR TITLE
Updated azure cosmos code owners with team name

### PR DESCRIPTION
**Pull request by kushagraThapar**
Updated azure cosmos code owners with team name

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -341,7 +341,7 @@
 # ServiceOwners:                                     @kushagraThapar @TheovanKraay @pjohari-ms
 
 # PRLabel: %Cosmos
-/sdk/cosmos/                                         @kushagraThapar @FabianMeiswinkel @kirankumarkolli @xinlian12 @milismsft @aayush3011 @simorenoh @jeet1995 @Pilchie @nehrao1
+/sdk/cosmos/                                         @kirankumarkolli @azure-cosmos-java-sdk-connectors
```
Comment (nehrao1) [inline .github/CODEOWNERS:344]: @kushagraThapar I think it needs to be:

@Azure/azure-cosmos-java-sdk-connectors

Comment (srnagar) [reply .github/CODEOWNERS:344]: yes, update this to use `@Azure/azure-cosmos-java-sdk-connectors `

Comment (kushagraThapar) [reply .github/CODEOWNERS]: Updated, thanks :)

**Comment by weshaggard** [comment]
I just approved @Azure/azure-cosmos-java-sdk-connectors to be a child of azure-sdk-write. However the cache only refreshes daily so retry the linter pipeline tomorrow and this should be good.

**Review by weshaggard — APPROVED** [review]

**Comment by kushagraThapar** [comment]
Thank you @weshaggard , appreciate it.

**Comment by kushagraThapar** [comment]
> I just approved @Azure/azure-cosmos-java-sdk-connectors to be a child of azure-sdk-write. However the cache only refreshes daily so retry the linter pipeline tomorrow and this should be good.

@weshaggard on the same note, can you please also approve this team - this is for our cosmos db python SDK https://github.com/orgs/Azure/teams/azure-cosmos-python-sdk

**Comment by weshaggard** [comment]
@kushagraThapar I don't see that request. I do see another request from you but it is for your own account. Perhaps you can try again?

**Review by srnagar — APPROVED** [review]

**Comment by kushagraThapar** [comment]
> @kushagraThapar I don't see that request. I do see another request from you but it is for your own account. Perhaps you can try again?

May be its in the process :)

image

**Comment by kushagraThapar** [comment]
> @kushagraThapar I don't see that request. I do see another request from you but it is for your own account. Perhaps you can try again?

Please ignore the one for my account, that was miss click :)

**Comment by weshaggard** [comment]
Found it and approved.

**Comment by kushagraThapar** [comment]
> Found it and approved.

Thank you :)

**Pull request merged**